inc/WireCellGenKokkos/BinnedDiffusion_transform.h

```diff
@@ -1,5 +1,5 @@
-#ifndef WIRECELL_GENKOKKOS_BINNEDDIFFUSION_TRANSFORM
-#define WIRECELL_GENKOKKOS_BINNEDDIFFUSION_TRANSFORM
+#ifndef WIRECELLGEN_BINNEDDIFFUSION_TRANSFORM
```

Comment:
This needs to be `WIRECELL_GENKOKKOS_BINNEDDIFFUSION_TRANSFORM` to avoid conflict with [this file](https://github.com/WireCell/wire-cell-toolkit/blob/master/gen/inc/WireCellGen/BinnedDiffusion_transform.h#L1)